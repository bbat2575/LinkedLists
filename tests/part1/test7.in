NEW 3
1
{
3
NEW 3
1
}
3
NEW 3
1
{}
3
NEW 3
1
 {
3
NEW 3
1
 }
3
NEW 3
1
 {}
3
NEW 3
1
a {
3
NEW 3
1
{a}
3
NEW 3
1
{ 1}
3
NEW 3
1
{1 }
3
NEW 3
1
{ 1 }
3
VIEW ALL
