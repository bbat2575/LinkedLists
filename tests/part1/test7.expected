
Type HELP for a list of options
Press <CTRL + D> to Quit

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> Number of lists: 0

> 
